locale/templates/settings.ftl

```diff
@@ -0,0 +1,407 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Globally reused strings
+
+# Note: the company name, Mozilla, is not translated.
```

Comment:
If we call out the names that shouldn't be translated, let's include Firefox as well.